packages/cli/src/commands/models/m365Sideloading.ts

```diff
@@ -10,7 +10,7 @@ export const sideloadingServiceEndpoint =
   process.env.SIDELOADING_SERVICE_ENDPOINT ?? serviceEndpoint;
 
 export const m365SideloadingCommand: CLICommand = {
-  name: "sideloading",
```

Comment:
What's the requirement to rename this? Or, is it possible to mark `sideloading` as deprecated but can still be executed, to keep backward compatibility?

There's already existing projects using `teamsfx m365 sideloading ...`, e.g., office-addin, MOS test, documentation... This rename is a breaking change and may break those projects.

Reply:
Offline discussed. This is for CLI v3 only. The whole CLI v3 is a breaking change so just ensure dependents and docs are upgraded as well.